vm/README.md

```diff
@@ -26,3 +26,8 @@ Virtual Machine
 MySQL Database
 * username = `root` 
 * password = `fpdetective`
+
+Some points that can help you to use this marvellous crawler with GUI,
```

Comment:
Thanks for calling this old crawler marvelous, but, let's drop this adjective as some may mistook it for self-flattery :-)
Can you resubmit after `s/this marvellous crawler/fpdetective/`
Otherwise I'll take care.